plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/messages.properties

```diff
@@ -19,6 +19,15 @@ stop.previous.version=Stop previous version
 use.custom.bucket=Override default Cloud Storage bucket for staging
 use.custom.versioning=Override generated version
 
+# Flex deploy settings
+browse.button=Browse
+config.folder.browse.button.message=The App Engine configuration folder is usually called \"appengine\"
+config.folder.browse.button.text=Select the App Engine configuration folder
+config.folder.location=App Engine configuration folder:
+docker.file.browse.button.text=Select the docker file
+docker.file.location=Docker file location:
+use.config.values=Use these values during deployment
```

Comment:
If these values have defaults, I'd prefer something similar like: "Override default values" so the wording is similar to the standard deploy UI.